tests/resources/existing-resources/outputs.tf

```
@@ -58,3 +58,13 @@ output "cluster_name" {
   value       = var.create_ocp_cluster ? module.ocp_base[0].cluster_name : null
   description = "The name of the provisioned cluster."
 }
+
+output "elasticsearch_crn" {
+  value       = module.elasticsearch.crn
+  description = "Elasticsearch CRN."
+}
+
+output "kms_instance_crn" {
+  value       = module.key_protect.key_protect_id
```

```suggestion
  value       = module.key_protect.key_protect_crn
```
[key protect CRN](https://github.com/terraform-ibm-modules/terraform-ibm-kms-all-inclusive/blob/829cd0b01a7654e0647898fc7cb40fbb885a51f0/outputs.tf#L25C9-L25C24) should be used as key_protect_id will provide service instance id.